pkg/packing/packer.go

```diff
@@ -28,6 +28,11 @@ import (
 	"k8s.io/apimachinery/pkg/api/resource"
 )
 
+const (
+	// maxInstanceTypes defines the number of instance type options to return to the cloud provider
+	maxInstanceTypes = 20
```

Comment:
Consider making this value public so that cloud providers can override it as necessary.

Reply:
good idea